Create ui-c8y-1018-503-81-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md (#1990)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-81-navigator-nodes-are-now-correctly-removed-when-refreshing-the-sub-assets.md

---
date: ""

> title: Groups in the navigator and subassets view now refresh correctly after removing groups via API

Reviewer: ```suggestion
title: Group navigation tree in navigator now refreshes correctly after removing groups via subassets view
```

> title: Navigator nodes are now correctly removed when refreshing the sub-assets grid after removing groups via API. [GRAFT][release/y2024] (#6205)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Sub-assets grid refreshes correctly after removing groups via API
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58562
version: 1018.503.81
---

> Navigator nodes are now correctly removed when refreshing the sub-assets grid after removing groups via API. [GRAFT][release/y2024] (#6205)

Author: The Claude AI suggested to use the following body:
```suggestion
In the Cumulocity IoT user interface, the sub-assets grid displays the assets that are part of a group. Previously, when removing a group via the API, the sub-assets grid was not updated correctly and still showed the now removed group assets. This has been fixed. Now, after removing a group via the API, refreshing the sub-assets grid will correctly remove any navigator nodes of assets that belonged to the deleted group, providing an up-to-date view of the group's assets in the user interface.
```

> In the Cumulocity IoT user interface, the subassets view and the navigator did not always update correctly after removing a group of devices via the API. This caused the navigator to still show the removed devices and made the UI inconsistent with the actual API state. With this fix, the subassets view and the navigator now correctly refresh after a group of devices is removed through the API, ensuring the user interface stays in sync and provides an accurate view of the current inventory.

Reviewer: ```suggestion
In the Cumulocity IoT user interface, the navigator did not always update correctly after removing a group or device via the subassets view. This caused the navigator to still show the removed groups or devices and made the UI inconsistent. With this fix, the navigator now correctly refreshes after a group or device has been removed through the subassets view, ensuring the user interface stays in sync and provides an accurate view of the current inventory.
```
